app/user/rest.py

```diff
@@ -454,6 +457,26 @@ def send_support_email(user_id):
     return jsonify({"status_code": status_code}), 204
 
 
+@user_blueprint.route('/<uuid:user_id>/contact-request', methods=['POST'])
```

Comment:
Don't think we can add this controller here: the contact form can be filled when you're logged out, we don't have a user ID in this case.

Reply:
That's strange because I've done most of my manual testing as a logged out user...
This field is filled - I will look into this

Reply:
Tested this with a logged out user and a logged in user the contact us page and it works
This also aligns with the previous implementation of `send_support_email` - marking it as resolved